tests/testthat/test-as_flex_table.R

```diff
@@ -1,5 +1,5 @@
 skip_on_cran()
-skip_if_not(is_pkg_installed("flextable"))
```

Comment:
Can you change this back, because the `is_pkg_installed()` function also checks the min required version specified in the DESCRIPTION file?